lib/router.js

```diff
@@ -88,15 +88,22 @@ router.close = function() {
   server = null;
 };
 
+router._routes = { };
```

Comment:
I defer to the consistency argument. In an ES6 world where there is a proper `Map` I would push for using that though. There's no reason for abusing objects in ES6 any more.